src/fipp/engine.cljc

```diff
@@ -5,6 +5,12 @@
             [fipp.deque :as deque]))
 
 
+(defn ^{:private true} unchunk [s]
```

Comment:
There's also this ticket: http://dev.clojure.org/jira/browse/CLJ-1218 - Is this faster maybe?